pkg/kubectl/cmd/get.go

```diff
@@ -315,5 +339,5 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			continue
 		}
 	}
-	return utilerrors.Flatten(utilerrors.NewAggregate(allErrs))
+	return utilerrors.NewAggregate(allErrs)
```

Comment:
You split the code into two chunks (one that uses Visit, and one that doesn't).  The previous code captured errors - nothing prevents them from being printed out without changing the block above to use Visit().